### ISSUE TITLE
Influence while offline (Ch1 PQ Destro-side)

### ISSUE THREAD

**Issue by mandersen55** [WarEmu/WarBugs#6772]
Greetings
I logged off at the crossroads just outside of the first pq on destro side.
RoR being as awsome as it is, I decided to log back on 10 minutes later (:
When i logged off, i had halfways to first dot on the influence bar. When i logged back on, I had completed CH1 influence.
Thinking that it was not intended, i asked in advice chat what i should do in this circumstance, and got a rather scary reply from someone mentioning that i had been reported for exploitation. I then asked a GM what to do, and he assured me that I would not get a ban, as it surely was unintended, as long as i posted a bugreport. So here I am. I am/was a lvl 5 Black guard by the name of Vitir, I was not grouped prior, nor after the logout.

Here is a couple of screenshots showing the conversation:

![vitir_000](https://cloud.githubusercontent.com/assets/19876928/15986656/3fe0bd84-300d-11e6-9b2e-b6b21fc70b6b.jpg)
![vitir_001](https://cloud.githubusercontent.com/assets/19876928/15986657/3fe148b2-300d-11e6-861e-17d97df15c48.jpg)
![vitir_003](https://cloud.githubusercontent.com/assets/19876928/15986655/3fe0a844-300d-11e6-88c2-e7e5d6c65de1.jpg)

I do not know how it happened, and with 6 or 7 characters completed ch1 pq influence, this is the first time i have encountered this bug, so I am affraid I cannot replicate it for further investigation

Regards
Vitir

**Comment by avenflar** [comment]
Were you able to pick the rewards at the rally master?

**Comment by mandersen55** [comment]
I was, I tried picking them once, and was unable to.
Then I decided to go back to the PQ to see if it was a display bug, where I didn't get influence for a "Militiamen" kill. I decided to go back to the rally master to check again, and was able to get the rewards the second time around. (The first time, the rewards was not grayed out, I could mark them, but they did not show up in my inventory, nor did they disable my choice for rewards.
